devices/salus_controls.js

```diff
@@ -86,6 +86,20 @@ module.exports = [
         exposes: [e.contact(), e.battery_low(), e.tamper()],
         ota: ota.salus,
     },
+    {
+        zigbeeModel: ['SS909ZB'],
+        model: 'PS600',
+        vendor: 'Salus Controls',
+        description: 'Pipe temperature sensor',
+        fromZigbee: [fz.temperature],
+        toZigbee: [],
+        exposes: [e.temperature(), e.battery_low(), e.tamper()],
```

Comment:
There are no fromZigbee converters which expose battery low and tamper so these should be removed or a converter should be added to fromzigbee